application/backend/src/main/resources/apps/typerefinery/components/graphs/barchart/head-libs.html

```diff
@@ -1 +1,2 @@
-<script src="/apps/typerefinery/components/graphs/barchart/script.js"></script>
\ No newline at end of file
+<script src="/apps/typerefinery/components/graphs/barchart/clientlibs/behaviour.js"></script>
+<script src="/apps/typerefinery/components/graphs/barchart/clientlibs/functions.js"></script>
```

Comment:
Please check these imports, When I tried this order of importing I faced some issues. 

Please verify the order of imports again.

Reply:
Done

Reply:
Please verify the imports.

If it is not resloved, please dont close that

Sample from linechart
![image](https://user-images.githubusercontent.com/54505967/216299955-d68029a4-7af6-4975-a3a3-21d620ea53b8.png)